src/drivers/display/drm/lv_linux_drm.c

```diff
@@ -836,4 +846,13 @@ static void drm_flush(lv_display_t * disp, const lv_area_t * area, uint8_t * px_
     }
 }
 
+static uint32_t tick_get_cb(void)
+{
+    struct timeval tv_now;
+    gettimeofday(&tv_now, NULL);
+    uint64_t time_ms;
+    time_ms = (tv_now.tv_sec * 1000000 + tv_now.tv_usec) / 1000;
+    return time_ms;
+}
```

Comment:
@kisvegabor please see this: #6296 :slightly_smiling_face: 

`clock_gettime(CLOCK_MONOTONIC, &period);` gets the time from a clock whose value doesn't jump when the system time changes. Can we use that here?

Reply:
Updated